pkg/controller/common/labels/labels.go

```diff
@@ -41,3 +39,8 @@ func (l TrueFalseLabel) AsMap(value bool) map[string]string {
 func AddCredentialsLabel(original map[string]string) map[string]string {
 	return maps.Merge(map[string]string{credentialsLabel: "true"}, original)
 }
+
+// New will return the common Elastic labels for any instance of the Elastic Stack.
+func New(obj commonv1.HasElasticLabels) (labels map[string]string) {
```

Comment:
I am not sure I see the point of this function (it is used only in one place where we could call the method from the interface directly as far as I can tell). It reads a tiny bit nicer because `labels.New` is pretty self-explanatory but maybe not worth it?

Reply:
Yeah, makes perfect sense.  Removed.  

*Note* I kept the test in this package, but renamed it to `TestGetIdentityLabels`, which tests all of the CRD's methods. It makes more sense from a package standpoint for this to go in `pkg/apis/common/v1`, but that would be an import cycle, so I left it here. The only other option would be to make 7 tests, one in each of the `pkg/apis/CRD/*` packages, which seemed overkill. Lmk your thoughts on that.